performance/graphite-setup.yaml

```diff
@@ -0,0 +1,102 @@
+dse_version: 6.0.11
+# Driver branch to use
+driver_oss_branch: 4.x
+# Driver dse branch to use
+driver_dse_branch: 4.x
+# Driver version identifier (used as part of graphite prefix)
+driver_version: 4.8.0
+# Driver examples branch to use
+driver_examples_branch: java-driver-4.x
+# How long to run test for
+duration: 2d
+
+---
+
+ensemble:
+  observer:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        kill_cluster_after_test: false
+        cloud.provider: openstack
+        cloud.tenant: performance
+        cloud.instance.type: ms1.small
+    configuration_manager:
+      - name: ctool_monitoring
+        properties:
+          graphite.create_server: true
+  server:
+    node.count: 3
+    provisioner:
+      name: ctool
+      properties:
+        kill_cluster_after_test: false
+        cloud.provider: openstack
+        cloud.tenant: performance
+        cloud.instance.type: ms1.small
+    configuration_manager:
+      - name: ctool
+        properties:
+          product.type: dse
```

Comment:
I wonder if we should switch to OSS C*?

Reply:
It is not important here. This is the only backend for grafana. It can be DSE or OSS C*

Reply:
C* clusters start much faster. But OK to leave DSE, who knows if we are adding Graph or Continuous Paging benchmarks one day.